pkgs/applications/misc/zettlr/default.nix

```diff
@@ -1,13 +1,16 @@
-{ appimageTools, lib, fetchurl, gtk3, gsettings-desktop-schemas}:
+{ appimageTools, lib, fetchurl, gtk3, gsettings-desktop-schemas
+, texlive, pandoc, pandoc-citeproc, corefonts
+, fonts ? [ corefonts ]
```

Comment:
This looks strange to me. Can someone else verify that this is good to merge?

Reply:
I think it is to support more export formats:

> Simple and beautiful exports with Pandoc, LaTeX, and Textbundle

https://github.com/Zettlr/Zettlr/blob/develop/README.md#features

Reply:
Yeah it is (read the rest of discussion above: https://github.com/NixOS/nixpkgs/pull/106074#issuecomment-739573401).

Reply:
Yes. And `core-fonts` is because when  Zettlr exports to PDF it sets a font not available inside the AppImage environment that NixOS create.

Reply:
@SuperSandro2000 I'm not happy with it either, but it seems like the appimage runs in some kind of chroot or sandbox and doesn't finde any system fonts. I'd be happy to adjust this PR if somebody tells me how that could be achieved for `appImageTools` (I really only wanted to bump this package, never used appImageTools before).

Reply:
I don't any better solution right now either.